tests/test_dmcli.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env pytest -vs
+"""Tests for Domain Manager CLI."""
+
+# Standard Python Libraries
+import sys
+from unittest.mock import patch
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+import dmcli
+from dmcli import main
+
+PROJECT_VERSION = dmcli.__version__
+
+
```

Comment:
It would be nice if there were some tests here specific to DM CLI, but I will still approve as long as you create an issue for this.

Reply:
@dav3r Yeah, we have some tasks in JIRA that cover this. At the moment, we've been focusing our tests on the API side and that's why this is not developed yet. Here is the issue I created in Github for it. https://github.com/cisagov/domain-manager-cli/issues/76